pkg/problemdaemon/custom_plugin_monitor.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package problemdaemon
+
+import (
+	"k8s.io/node-problem-detector/pkg/custompluginmonitor"
+	"k8s.io/node-problem-detector/pkg/types"
+)
+
+func init() {
+	register(
```

Comment:
Make this public, and move into its own corresponding package?
Basically keep all custom plugin monitor logic in its own package.

Reply:
Sounds good. Thanks!